docs/user/exams/instructors_guide.rst

```diff
@@ -70,7 +88,23 @@ During the exam creation and configuration, you can create your exam and configu
 
    Exam Assessment and Student Review
 
-    .. _exercise_groups:
+
+
+**1.2.3 Import Exam**
+^^^^^^^^^^^^^^^^^^^^^^
+
+- Alternatively, you can import an existing exam by clicking on |import_exam|.
```

Comment:
The section starts with ‘Alternatively, […]’. At that point it is not obvious as alternative to what.

```suggestion
- Instead of creating a new exam, you can import an existing exam by clicking on |import_exam| from any of the courses you are an instructor/editor (?) in.
```